### DISCUSSION TITLE
How do we default the IOS Simulator to X64 using Visual Studio Code with .NET 8

### DISCUSSION THREAD

**Discussion by agingo** [dotnet/maui#20349] [General]
We've created a .NET Core 8 MAUI application for iOS that uses custom C++ libs that are compiled for IOS-ARM64 and IOSsimulator-X64 using vcpkg. 

Since moving to an Apple M2 with .NET 8 when we start debugging from Visual Studio Code it defaults to using an ARM64 Simulator. 

Does anyone know how to change Visual Studio Code so the IOS Simulator defaults to X64?

**Comment by xanril** [comment]
I haven't tried this in .NET 8, but see this thread and comments  in microsoft/vscode-dotnettools#744 .
The issue should be addressed in preview 0.8.3. I've added `<ForceSimulatorX64ArchitectureInIDE>true</ForceSimulatorX64ArchitectureInIDE>` in my `.csproj` and vscode builds it for `iossimulator-x64`
